packages/compass-aggregations/src/modules/aggregation.ts

```diff
@@ -204,9 +224,109 @@ const reducer: Reducer<State> = (state = INITIAL_STATE, action) => {
   return state;
 };
 
+const isConfirmReceivedOrNotRequired = async (
```

Comment:
nit: what about calling this just `confirmWriteOperation`? I was thinking that for someone without context, knowing that this is for write operations at call point can be valuable piece of info. I would be able to infer that is ignoring non "write" pipelines. Otherwise we can also say `confirmWriteOperationIfNeeded`

Reply:
I will update with `confirmWriteOperationIfNeeded` 👍